package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
```

Comment:
@IgorChaikin Could you also create `.npmrc` file with `package-lock=false` and remove `package-lock.json` from git